relentless/potential/potential.py

```diff
@@ -2,6 +2,7 @@
 
 import json
 import numpy as np
+import abc
```

Comment:
Sorry, forgot to also mention: please add imports in alphabetical order, so `abc` would come before `json`.